WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
 	# Extra income due to donations, etc
 	monthly_income = 10
 
+}
+
+c_priscillianism_holy_order = {
```

Comment:
That was intentional, so that the Priscillian Pope could vassalize them, or the player could transfer them to the Priscillian Pope. The Priscillian Pope should be King-tier, but I stopped disputing that a long time ago.